.github/workflows/documentlint.yml

```diff
@@ -0,0 +1,35 @@
+name: "Lint document files"
+
+on:
+  push:
+    branches:
+      - main
+      - ipdk_v*
+  pull_request:
+    branches:
+      - main
+      - ipdk_v*
+  workflow_dispatch:
+
+concurrency:
+  # If workflow for PR or push is already running, stop it and start a new one.
+  group: build-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions: read-all
+
+jobs:
+  documentlint:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+
+    steps:
+      - name: Clone networking-recipe
+        uses: actions/checkout@v4
+        with:
+          path: recipe
+
+      - name: Lint markdown files
+        uses: nosborn/github-action-markdown-cli@v3.3.0
+        with:
+          files: docs/**/*.md
```

Comment:
How about .rst files? Does this action support rst files?

Also, does the

Reply:
No, the markdownlint CLI does not support rst files.

Reply:
I've added jobs to run `doc8` and `rstcheck` on restructuredtext (.rst) files.